syck.h

```diff
@@ -1,10 +1,4 @@
 #include "config.h"
-#ifdef I_STDLIB
-#define HAVE_STDLIB_H
```

Comment:
This is not needed any more, right?

Reply:
Yes. My grep does not find any HAVE_STDLIB_H in the YAML-Syck sources.